hack/verify-all.sh

```diff
@@ -21,4 +21,3 @@ PKG_ROOT=$(git rev-parse --show-toplevel)
 ${PKG_ROOT}/hack/verify-gofmt.sh
 ${PKG_ROOT}/hack/verify-govet.sh
 ${PKG_ROOT}/hack/verify-golint.sh
-${PKG_ROOT}/hack/verify-dep.sh
```

Comment:
is there similar way to verify go module?

Reply:
we don't need any check for go modules I think as it will fetch the required automatically during the build process.

Reply:
> @ashishranjan738 a single go mod tidy is not enough. see https://github.com/kubernetes/cloud-provider-azure/blob/master/hack/update-dependencies.sh for a reference.

cool will do it